src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,39 @@
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+interface Props {
```

Comment:
[Not critical]
It's better to name this type something like `TodoItemProps`, so that the name describes this type the most precisely.